### DISCUSSION TITLE
Table reload after open side panel

### DISCUSSION THREAD

**Discussion by fadliasyhariStar** [Budibase/budibase#12748] [Help] [answered]
There's a different behavior between screens. When I try to open the side panel from page 2, the table will reload and back to page 1, and it only happens on 1 screen whereas the actions are the same as other screens. It's pretty difficult to troubleshoot this.

![Screencast from 10-01-24 11_08_09](https://github.com/Budibase/budibase/assets/134992316/7bd9a403-e564-44f8-8683-bfbd16619da9)

I'm using version 2.13.9, self-hosted, and PostgreSQL data sources.
The reason why I still use version 2.13.9 is once I tried to update the version, one of the screens was broken(screen /person/detail/:id). It just shows a blank screen and I cannot edit the components on the sides. I cannot find which components that I use incorrectly on this screen. Please help me with this also. Thanks in advance.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @fadliasyhariStar,

I removed the app export you provided as it included your live Postgres database even though It seemed to be filled with test data i thought it best to remove from your post.

The issue you are experiencing lies with this filter. This gets re-run as the date has changed which causes the pagination to be reset back to one. To stop this remove this filter and it will work normally again. This is like to do with how dates are being outputted, its re-ran after opening the side panel and hitting another document data provider causing the original to also be re-run as the date as been updated.

<img width="891" alt="Screenshot 2024-01-10 at 13 28 31" src="https://github.com/Budibase/budibase/assets/126772285/78113092-fb7b-4dbd-ac17-0b6589303f3f">

**Reply by fadliasyhariStar** [reply]
Thank you for the answer, do you have another way to do a between-date filter but not using a custom query?
